.github/workflows/performance_metrics_cron.yml

```diff
@@ -98,7 +103,22 @@ jobs:
           echo "PULUMI_TRACING_TAG_PULUMI_VERSION=$(pulumi version)" >> $GITHUB_ENV
           echo "PULUMI_TRACING_DIR=$PWD/traces"                      >> $GITHUB_ENV
       - name: Running Performance Matrix
-        run: make test_templates
+        shell: bash
```

Comment:
I've not seen this specified before. Is Bash not the default?

Reply:
I think not - the other code had it but I'm not sure.